### ISSUE TITLE
Add SSL section

### ISSUE THREAD

**Issue by drAlberT** [h5bp/server-configs-apache#72]
It would be fantastic to have an optimised/secured SSL section:
- disable insecure cyphers
- disable insecure protos
- enable/disable renegotiation

and so on ..

**Comment by LeoColomb** [comment]
Could you elaborate? Maybe make a PR?
But I wonder if it's not too certificate-specific...

**Comment by drAlberT** [comment]
Sorry I don't use apache anymore on my servers..

I was talking about SSL general settings, nothing related to specific cert ..

I was thinking about something as following:
```
SSLProtocol      all -SSLv3 -TLSv1 -TLSv1.1
SSLCipherSuite   ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256
```

Refs:
- http://httpd.apache.org/docs/trunk/ssl/ssl_howto.html
- https://mozilla.github.io/server-side-tls/ssl-config-generator/?server=apache-2.4.0&openssl=1.0.1e&hsts=yes&profile=modern

**Comment by LeoColomb** [comment]
Could be interesting to combine with the [Directive `SSLPolicy`](https://httpd.apache.org/docs/trunk/mod/mod_ssl.html#sslpolicy), available first in httpd 2.4.28.
It will be easier and more user-friendly.
